.github/reusable_wfs/setup.yml

```
@@ -0,0 +1,30 @@
+name: Checkout commit and build for tests
+
+on:
+  workflow_call:
+
+jobs:
+  setting_up:
+    runs-on: ubuntu-latest
+    container: node:14
+    
+    steps:
+      - name: check event type
+        id: evt
+        run: |
+          if [ ${{ github.event_name }} == 'pull_request_target' ]; then
+            echo "::set-output name=ref::refs/pull/${{ github.event.number }}/merge"
+          elif [ ${{ github.event_name }} == 'workflow_dispatch' ]; then
+            echo "::set-output name=ref::${{ github.event.ref }}"
+          fi
+        shell: bash
+
+      - uses: actions/checkout@v2
```

Can this be updated to use the latest version of the action?

```suggestion
      - uses: actions/checkout@v3
```